pkg/controllers/managementuser/rbac/handler_base.go

```diff
@@ -570,8 +570,6 @@ func getPRTBProjectAndSubjectKey(prtb *v3.ProjectRoleTemplateBinding) string {
 		name = prtb.GroupName
 	} else if prtb.GroupPrincipalName != "" {
 		name = prtb.GroupPrincipalName
-	} else if prtb.ServiceAccount != "" {
```

Comment:
I don't think that we should be removing library-type items like this. It reduces the ability of our code to manage existing items, particularly in the case of a remove event.

Put another way, if I delete an active PRTB bound to a service account, with this function changed, how is the RBAC code going to identify downstream bindings?